mlkem/mlkem_native.h

```diff
@@ -5,76 +5,50 @@
 
 #ifndef MLK_H
 #define MLK_H
-/*
+
+/******************************************************************************
+ *
  * Public API for mlkem-native
  *
  * This header defines the public API of a single build of mlkem-native.
  *
- * To use this header, make sure one of the following holds:
+ * # Usage
+ *
+ * To use this header, configure the following options:
+ *
+ * - MLK_CONFIG_API_PARAMETER_SET [required]
+ *
+ *   The parameter set used for the build; 512, 768, or 1024.
+ *
+ * - MLK_CONFIG_API_NAMESPACE_PREFIX [required]
+ *
+ *   The namespace prefix used for the build.
+ *
+ *   NOTE:
+ *   For a multilevel build, you must include the 512/768/1024 suffixes
+ *   in MLK_CONFIG_API_NAMESPACE_PREFIX.
+ *
+ * - MLK_CONFIG_API_NO_SUPERCOP [optional]
+ *
+ *   By default, this header will also expose the mlkem-native API in the
+ *   SUPERCOP naming convention crypto_kem_xxx. If you don't want/need this,
+ *   set MLK_CONFIG_API_NO_SUPERCOP. You must set this for a multilevel build.
  *
- * - The config.h used for the build is available in the include paths.
- * - The values of MLK_BUILD_INFO_LVL and MLK_BUILD_INFO_NAMESPACE are set,
- * reflecting the security level (512/768/1024) and namespace of the build.
+ * - MLK_CONFIG_API_CONSTANTS_ONLY [optional]
+ *
+ *   If you don't want this header to expose any function declarations,
+ *   but only constants, set MLK_CONFIG_API_CONSTANTS_ONLY. In this case,
```

Comment:
should we elborate here that the constants are refering to public key/ciphertext/etc sizes? constants is a bit vague

Reply:
Ok